src/fontra/client/core/var-glyph.js

```diff
@@ -56,6 +56,7 @@ export class StaticGlyph {
     }
     source.components =
       (noCopy ? obj.components : obj.components?.map(copyComponent)) || [];
+    source.anchors = obj.anchors || [];
```

Comment:
You need to deep copy the anchors (unless `noCopy` is true). You can use the `copyCustomData()` function for that, which is a little poorly named once we use it for anchors.

Reply:
Thanks. Done.